Dockerfile

```diff
@@ -1,17 +1,39 @@
-FROM node:12.18-alpine
-
+# Build image
+FROM node:12.18-alpine AS build
 ARG DATABASE_TYPE
-
 ENV DATABASE_URL "postgresql://umami:umami@db:5432/umami" \
     DATABASE_TYPE=$DATABASE_TYPE
+WORKDIR /build
 
-WORKDIR /app
-EXPOSE 3000
+COPY package.json yarn.lock /build/
 
-COPY package.json yarn.lock /app/
-RUN yarn install --frozen-lockfile
+# Install only the production dependencies
+RUN yarn install --production
```

Comment:
better to keep `--frozen-lockfile`

@hugomd

Reply:
Good catch, I thought I included it but I guess it got lost somewhere along the way! I'll make a PR to re-add it 👍🏻